src/app/helpers/tests/testHelpers.js

```diff
@@ -1,11 +1,11 @@
-import renderer from 'react-test-renderer'; // eslint-disable-line import/no-extraneous-dependencies
 import 'jest-styled-components'; // eslint-disable-line import/no-extraneous-dependencies
 import ShallowRenderer from 'react-test-renderer/shallow'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
What's the reason for keeping the shallow render? Surely we want to remove all uses of `react-test-renderer` and move over to using `react-testing-library`?

Reply:
It's currently for the Metatdata component, which uses Helmet and so the component itself doesn't have any rendered DOM nodes.  We could use assertion testing rather than snapshot testing for this component. I will look into that and add it to this PR.

Reply:
Ahhh that makes sense, yeah that would be good. Myself and @bcmn did some assertion testing for the Metadata container [which could potentially be extended](https://github.com/BBC-News/simorgh/blob/latest/src/app/containers/Metadata/index.test.jsx#L120-L124)